api/app/reviews/api.py

```diff
@@ -124,8 +130,10 @@ def get(self):
         skip = self.sanitise_skip(args['skip'], reviews_remaining_count)
 
         response = review_repository.get_response_to_review(skip, g.current_user['id'], review_form.application_form_id)
+        print(response)

```

Comment:
Good morning, @avishkar58 ! I've been sitting with this for a few days now and I'm out of ideas. The `response` is returning `None` and I'm not sure where the error is!

Reply:
Hey @SMHall94 ! So the reason for this is because the test it's requesting a response for "firstuser" (the logged in user) to review, but "firstuser" doesn't have any responses assigned to them. You can add a ResponseReviewer in your seed_static_data method to assign firstuser a response to review - see [here](https://github.com/deep-learning-indaba/Baobab/blob/6c9b8d4d2051273ca23493d7012b18710f6f0839/api/app/reviews/tests.py#L493) for example. The two IDs passed to the constructor are the response_id and the reviewer_user_id, so you can do something like this in your seed_static_data method:

```
response_review = ResponseReviewer(self.test_response.id, self.first_user.id)
db.session.add(response_review)
db.session.commit()
```